packages/blockchain-sdk-aptos/src/wallet-adapter/node-dialect-aptos-wallet-adapter.ts

```diff
@@ -30,7 +30,7 @@ export class NodeDialectAptosWalletAdapter
       );
       return new NodeDialectAptosWalletAdapter(account);
     }
-    if (process.env.DIALECT_SDK_CREDENTIALS) {
+    else if (process.env.DIALECT_SDK_CREDENTIALS) {
```

Comment:
is this code formatting issue?

Reply:
no this was an intended change

Reply:
it is now if, elif, else pattern -- prev just if, if return early

Reply:
@zhelezkov

Reply:
have run prettier, thx!